index.html

```diff
@@ -38,6 +39,204 @@ <h2 id="job-title">I am a Software Developer</h2>
+                            <a href=""><button class="project-1-buttons">html</button></a>
+                            <a href=""><button class="project-1-buttons" >bootstrap</button></a>
+                            <ahref=""><button class="project-1-buttons">Ruby</button></a>
+                        </div>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button id="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+                <div class="project-1">
+                    <a id="project-backdrop" href=""><img src="images/Img Placeholder-2.png" alt=""></a>
+                    <div class="floating">
+                        <div>
+                            <h3 id="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p id="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        </div>
+                        <div>
+                            <a href=""><button class="project-1-buttons">html</button></a>
+                            <a href=""><button class="project-1-buttons" >bootstrap</button></a>
+                            <ahref=""><button class="project-1-buttons">Ruby</button></a>
+                        </div>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button id="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+                <div class="project-1">
+                    <a id="project-backdrop" href=""><img src="images/Img Placeholder-2.png" alt=""></a>
+                    <div class="floating">
+                        <div>
+                            <h3 id="project-title">Profesional Art Printing Data</h3>
+                        </div>
+                        <div>
+                            <p id="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        </div>
+                        <div>
+                            <a href=""><button class="project-1-buttons">html</button></a>
+                            <a href=""><button class="project-1-buttons" >bootstrap</button></a>
+                            <ahref=""><button class="project-1-buttons">Ruby</button></a>
+                        </div>
+                    </div>
+                    <div class="see-project-div">
+                        <a href=""><button id="see-project-button">See Project</button></a>
+                    </div>
+                </div>
+                <div class="project-1">
+                    <a id="project-backdrop" href=""><img src="images/Img Placeholder-2.png" alt=""></a>
```

Comment:
Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image